### PR TITLE
Add link to data browser + prefix in dataset overview page

### PR DESCRIPTION
**Pull request by slesaad**
# Description
- Add a `download` button to the dataset overview page that links to the data browser > /path-to-collection/

# Demo
https://deploy-preview-41--ghg-demo.netlify.app/data-catalog/land-atmoshphere-co2-emissions-monthly-2022

### CODE REVIEW

**Comment by slesaad** [comment]
@danielfdsilva @hanbyul-here can you help make this better?

**Comment by danielfdsilva** [comment]
After some internal discussion we agreed that adding another button to the dataset header is detrimental to the user experience. We recommend to use the "Analyse data" modal to link to external sources and/or having a direct link in the dataset page content.

We will set this up as an example and change the "analyse data" button label to better reflect the new modal nature. cc @hanbyul-here

**Comment by slesaad** [comment]
@danielfdsilva @hanbyul-here let's hold this off for now, we showed a demo of this new feature, and it looks like we might need to make some changes. i think the external url will still hold, but how we do it might change.

**Comment by hanbyul-here** [comment]
I went ahead and made the link to data browser as a part of 'Access Data' Modal. I updated this pr on veda ghg accordingly: https://github.com/NASA-IMPACT/veda-config-ghg/pull/41/files

**Review by danielfdsilva — APPROVED** [review]

**Review by j08lue — CHANGES_REQUESTED** [review]
Tiny typo

**Review by j08lue — APPROVED** [review]

File: app/scripts/components/common/notebook-connect.tsx [hunk 1]
```diff
@@ -100,6 +102,8 @@ export function NotebookConnectModal(props: {
       let type = 'unknown';
       if (d.url.match('nasa-veda.2i2c.cloud')) type = 'jupyter';
       else if (d.url.match('github.com/NASA-IMPACT/veda-docs')) type = 'github';
+      // TO DO: browser UI url pattern check
+      else if (d.url.match('browseui')) type = 'download';
```
[the remark this answers is not in the document]
Comment (hanbyul-here) [reply app/scripts/components/common/notebook-connect.tsx:106]: (had a conversation with @slesaad and the answer is yes.

**Pull request merged**